src/main/java/edu/harvard/iq/dataverse/MailServiceBean.java

```diff
@@ -546,6 +546,49 @@ public String getMessageTextBasedOnNotification(UserNotification userNotificatio
                 logger.fine("fileImportMsg: " + fileImportMsg);
                 return messageText += fileImportMsg;
 
+            case GLOBUSUPLOADCOMPLETED:
+                dataset =  (Dataset) targetObject;
+                messageText = BundleUtil.getStringFromBundle("notification.email.greeting.html");
```

Comment:
Is there a reason you need the html greeting just for these or could/should we just pass the isHtmlContent param into this method and switch for all the notifications?